src/app/projects/projects-table/projects-table.component.css

```diff
@@ -3,4 +3,13 @@
     background-color: whitesmoke;
     margin: 10px 0 10px 0;
     min-height: 300px;
+    position: relative;
+}
+
+#new-project-button {
+    margin: 0;
+    position: absolute;
```

Comment:
nie jestem pewien, czy robienie czegoś `absolute` to dobry pomysł
raczej celowałbym w wyrównanie do prawej (o ile w takiej formie menubara się da - ale zawsze można odejść od tego oferowanego przez primeng)

Reply:
ten przycisk akurat nie ma nic wspólnego z menubarem. natomiast `absolute` na nim i `relative` na zewnętrznym divie pozwala na wyśrodkowanie w pionie (vertical align chyba się to nazywa)